.github/workflows/sentryRelease.yml

```diff
@@ -0,0 +1,22 @@
+name: ReleaseWorkflow
+
+on:
+  release:
+    types: [published, prereleased]
+
+
+jobs:
+  createSentryRelease:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+      - name: Create a Sentry.io release
+        uses: tclindner/sentry-releases-action@v1.2.0
+        env:
+          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+          SENTRY_ORG: eigengraus-generator
+          SENTRY_PROJECT: eigengraus-generator
+        with:
+          tagName: ${{ github.ref }}
+          environment: qa
```

Comment:
We could perhaps rename this to `production`

Reply:
I'm not quite up to date on Github Actions best practices- is file splitting for each function preferred, or should we concatenate based on the trigger? Because if we were to add something like an auto-update-poster, it'd make perfect sense to name it that, otherwise it's not really very descriptive of what the action does. Up to you- they're most definitely not my department.